apps/nfid-frontend/src/index.css

```diff
@@ -63,6 +63,10 @@ ul {
   color: #000;
 }
 
+.Toastify__toast-body {
```

Comment:
just in case, may be for mobile it makes sense to have width the same as other content? like almost 100%